From 3dc38b2547cb314bcf3d0a51b96a4f54680adca6 Mon Sep 17 00:00:00 2001
From: Ricky CX Wu <ricky.cx.wu.wiwynn@gmail.com>
Date: Thu, 1 Aug 2024 14:15:05 +0800
Subject: [PATCH 6/7] configurations: yosemite4: set fail safe percent to 90%
 for Fan fail

Gerrit: 73176.

Set fail safe percent to 90% for fan sensors so that pid-control service
would set fan PWM to 90% when one of the fan failed.

Note: Need to apply Gerrit 67566 before adding this change.

Tested:
Tested with patch of Gerrit 67566 and checked that the fan PWM would be
set to 90% when one fan failed.

Change-Id: I38acd26f2610ed0ae9dd95fc2130b124383b0be1
Signed-off-by: Ricky CX Wu <ricky.cx.wu.wiwynn@gmail.com>
---
 configurations/yosemite4.json  | 1 +
 configurations/yosemite4n.json | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configurations/yosemite4.json b/configurations/yosemite4.json
index 0c2d19b..4f57db0 100644
--- a/configurations/yosemite4.json
+++ b/configurations/yosemite4.json
@@ -438,6 +438,7 @@
             "Class": "fan",
             "FFGainCoefficient": 1.0,
             "FFOffCoefficient": 0.0,
+            "FailSafePercent": 90.0,
             "ICoefficient": 0.0,
             "ILimitMax": 0.0,
             "ILimitMin": 0.0,
diff --git a/configurations/yosemite4n.json b/configurations/yosemite4n.json
index 49a6769..a3f9e0a 100644
--- a/configurations/yosemite4n.json
+++ b/configurations/yosemite4n.json
@@ -412,6 +412,7 @@
             "Class": "fan",
             "FFGainCoefficient": 1.0,
             "FFOffCoefficient": 0.0,
+            "FailSafePercent": 90.0,
             "ICoefficient": 0.0,
             "ILimitMax": 0.0,
             "ILimitMin": 0.0,
-- 
2.44.2

